### PR TITLE
Automatize knmstate

### PR DESCRIPTION
**Pull request by qinqon**
**What this PR does / why we need it**:
As part of the future effort of automatize kubernetes-nmstate has to be included at the "autobump" script here.

To easy integration the kubernets-nmstate manifests at CNAO has being renamed to reduce automatization friction.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by phoracek** [comment]
@qinqon this enables the webhook again. Is it safe?

**Comment by qinqon** [comment]
@phoracek why did we disable it at CNV 2.4 ? leader election + upgrades right ?

**Comment by phoracek** [comment]
> @phoracek let enable again and see if upgrade works, also we have to change strategy to Recreate at knmstate [nmstate/kubernetes-nmstate#566](https://github.com/nmstate/kubernetes-nmstate/pull/566)

Then we should wait for that PR to get in before we enable it.

File: data/nmstate/operator.yaml [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+spec:
+  publishNotReadyAddresses: true
+  ports:
+    - port: 443
+      targetPort: 8443
+  selector:
+    name: {{template "handlerPrefix" .}}nmstate-webhook
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
```
Comment (phoracek) [inline data/nmstate/operator.yaml:186]: You need to list it under test check components https://github.com/kubevirt/cluster-network-addons-operator/blob/de15dd0aa3c79a38a21ccc82d0df900da054abad/test/check/components.go#L21

File: test/releases/99.0.0.go [hunk 2]
```diff
@@ -36,7 +36,7 @@ func init() {
 				ParentName: "nmstate-handler",
 				ParentKind: "DaemonSet",
 				Name:       "nmstate-handler",
-				Image:      "quay.io/nmstate/kubernetes-nmstate-handler:v0.21.0",
+				Image:      "quay.io/nmstate/kubernetes-nmstate-handler:v0.22.0",
```
Comment (phoracek) [inline test/releases/99.0.0.go:39]: You have to list the webhook too.

**Pull request merged**